e2e/content-services/document-list/document-list-selection.e2e.ts

```diff
@@ -0,0 +1,145 @@
+    });
+
+    this.alfrescoJsApi = new AlfrescoApi({
+        provider: 'ECM',
+        hostEcm: browser.params.testConfig.adf_acs.host
+    });
+
+    beforeAll(async () => {
+        await this.alfrescoJsApi.login(browser.params.testConfig.adf.adminEmail, browser.params.testConfig.adf.adminPassword);
+        await this.alfrescoJsApi.core.peopleApi.addPerson(acsUser);
+        await this.alfrescoJsApi.login(acsUser.id, acsUser.password);
+
+        await loginPage.loginToContentServicesUsingUserModel(acsUser);
+
+        await contentServicesPage.goToDocumentList();
+        await contentServicesPage.checkDocumentListElementsAreDisplayed();
+        await contentServicesPage.createNewFolder(folderModel.name);
+        await contentServicesPage.uploadFile(docxFileModel.location);
+        await contentServicesPage.checkContentIsDisplayed(docxFileModel.name);
+        await uploadDialog.clickOnCloseButton();
+        await uploadDialog.dialogIsNotDisplayed();
+    });
+
+    afterAll(async () => {
+        await navigationBarPage.clickLogoutButton();
+    });
+
+    it('[C274696] Should be able to select and unselect a file or folder', async () => {
+        await contentServicesPage.selectRow(docxFileModel.name);
+        await contentServicesPage.getDocumentList().dataTablePage().checkRowIsSelected('Display name', docxFileModel.name);
+
+        await paginationPage.clickItemsPerPageDropdown();
+        await contentServicesPage.getDocumentList().dataTablePage().checkRowIsSelected('Display name', docxFileModel.name);
+
+        await contentServicesPage.selectRow(docxFileModel.name);
+        await contentServicesPage.getDocumentList().dataTablePage().checkRowIsNotSelected('Display name', docxFileModel.name);
+
+        await contentServicesPage.selectRow(folderModel.name);
+        await contentServicesPage.getDocumentList().dataTablePage().checkRowIsSelected('Display name', folderModel.name);
+
+        await paginationPage.clickItemsPerPageDropdown();
+        await contentServicesPage.getDocumentList().dataTablePage().checkRowIsSelected('Display name', folderModel.name);
+
+        await contentServicesPage.selectRow(folderModel.name);
+        await contentServicesPage.getDocumentList().dataTablePage().checkRowIsNotSelected('Display name', folderModel.name);
+    });
+
+    it('[C260057] Should be able to choose between the Selection Mode options and select items accordingly', async () => {
+        let list;
+        await contentServicesPage.chooseSelectionMode('None');
+
+        await contentServicesPage.selectRow(docxFileModel.name);
+        await contentServicesPage.selectFolderWithCommandKey(folderModel.name);
+        await contentServicesPage.getDocumentList().dataTablePage().checkRowIsNotSelected('Display name', docxFileModel.name);
+        await contentServicesPage.getDocumentList().dataTablePage().checkRowIsNotSelected('Display name', folderModel.name);
```

Comment:
You can declare the Display name into a variable and avoid hardcoding it in many places. This will make easier to maintain this tests in the future by changing only the variable instead of every line that has this hardcoded string. You can call it rowName or something similar

Reply:
done